### ISSUE TITLE
Consider swapping uv for Poetry

### ISSUE THREAD

**Issue by jmgate** [sandialabs/reverse_argparse#191]
## Description
Experiment with swapping out [uv](https://github.com/astral-sh/uv) for Poetry in terms of environment management, building the project, etc.

**Comment by jmgate** [comment]
Here's an example of [how to make the switch](https://www.caktusgroup.com/blog/2025/06/11/migrating-python-django-projects-uv/).

**Comment by GhostofGoes** [comment]
What is the reason to swap?

**Comment by GhostofGoes** [comment]
Agreed. I misread the original issue and for some reason thought it was going from uv -> Poetry, not the other way around (it's been a hot minute since I've looking at the repo). +1
